behavior-modules/greet.js

```diff
@@ -0,0 +1,7 @@
+const Behavior = require('../models/behavior');
```

Comment:
Sorry my below comment was supposed to be for here. Your index.js idea is perfect.